tasks/edge.rake

```diff
@@ -7,10 +7,17 @@ require_relative 'runtime_matcher'
 namespace :edge do
   desc 'Update all the groups with gemspec dependencies'
   task :gemspec do |_t, _args|
+    major, minor, = Gem::Version.new(RUBY_ENGINE_VERSION).segments
+    ruby_runtime = "#{RUBY_ENGINE}-#{major}.#{minor}"
+
     candidates = Set.new
 
     TEST_METADATA.each do |_, metadata|
       metadata.each do |group, rubies|
+        # Exceptions:
+        # Rails 4.x is not supported on JRuby 9.2 (which is RUBY_VERSION 2.5)
+        next false if ruby_runtime == 'jruby-9.2' && group.start_with?('rails4')
+
```

Comment:
Hmm I wonder if this is the right place to do this... E.g. it looks like we're reinventing a bit the logic we already have in the `Matrixfile`.

E.g. Maybe we could flip the Rails 4 tests to `❌ jruby`?